package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "postcss-sorting",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

```suggestion
  "version": "5.0.0",
```

It will be changed with my publishing tool.